test/tests/parallel/set1/dovecot/dovecot_quotas.bats

```diff
@@ -0,0 +1,249 @@
+  assert_output '2M'
+
+  # Sizes are equivalent - Bytes to MiB (rounded):
+  run numfmt --to=iec --format '%.0f' 2048000
+  assert_output '2M'
+}
+
+@test 'Deleting an mailbox account should also remove that account from dovecot-quotas.cf' {
+  _add_mail_account_then_wait_until_ready 'quserremoved@domain.tld'
+
+  _run_in_container setup quota set quserremoved@domain.tld 12M
+  assert_success
+
+  _run_in_container cat '/tmp/docker-mailserver/dovecot-quotas.cf'
+  assert_success
+  assert_output 'quserremoved@domain.tld:12M'
+
+  _run_in_container setup email del -y quserremoved@domain.tld
+  assert_success
+
+  _run_in_container cat /tmp/docker-mailserver/dovecot-quotas.cf
+  assert_success
+  refute_output --partial 'quserremoved@domain.tld:12M'
+}
+
+@test 'Dovecot should acknowledge quota configured for accounts' {
+  # sed -nE 's/.*STORAGE.*Limit=([0-9]+).*/\1/p' | numfmt --from-unit=1024 --to=iec --format '%.0f'
+  local CMD_GET_QUOTA="doveadm -f flow quota get -u 'user1@localhost.localdomain'"
+
+  # 4M == 4096 kiB (numfmt --to-unit=1024 --from=iec 4M)
+  _run_in_container_bash "${CMD_GET_QUOTA}"
+  assert_line --partial 'Type=STORAGE Value=0 Limit=4096'
+
+  # Setting a new limit for the user:
+  _run_in_container setup quota set 'user1@localhost.localdomain' 50M
+  assert_success
+  # 50M (50 * 1024^2) == 51200 kiB (numfmt --to-unit=1024 --from=iec 52428800)
+  run _repeat_until_success_or_timeout 20 _exec_in_container_bash "${CMD_GET_QUOTA} | grep -o 'Type=STORAGE Value=0 Limit=51200'"
+  assert_success
+
+  # Deleting quota resets it to default global quota limit (`plugin/quota_rule`):
+  _run_in_container setup quota del 'user1@localhost.localdomain'
+  assert_success
+  run _repeat_until_success_or_timeout 20 _exec_in_container_bash "${CMD_GET_QUOTA} | grep -o 'Type=STORAGE Value=0 Limit=4096'"
+  assert_success
+}
+
+@test 'should receive a warning mail from Dovecot when quota is exceeded' {
+  # skip 'disabled as it fails randomly: https://github.com/docker-mailserver/docker-mailserver/pull/2511'
+
+  # Prepare
+  _add_mail_account_then_wait_until_ready 'quotauser@otherdomain.tld'
+
+  # Actual tests
+  _run_in_container setup quota set quotauser@otherdomain.tld 10k
+  assert_success
+
+  # wait until quota has been updated
+  run _repeat_until_success_or_timeout 20 _exec_in_container_bash "doveadm -f flow quota get -u 'quotauser@otherdomain.tld' | grep -o 'Type=STORAGE Value=0 Limit=10'"
+  assert_success
+
+  # dovecot and postfix has been restarted
+  _wait_for_service postfix
+  _wait_for_service dovecot
+  sleep 10
+
+  # send some big emails
+  _run_in_container_bash "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/quota-exceeded.txt"
+  assert_success
+  _run_in_container_bash "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/quota-exceeded.txt"
+  assert_success
+  _run_in_container_bash "nc 0.0.0.0 25 < /tmp/docker-mailserver-test/email-templates/quota-exceeded.txt"
+  assert_success
```

Comment:
These were changed since, carrying that over:

```suggestion
  _send_email 'email-templates/quota-exceeded' '0.0.0.0 25'
  _send_email 'email-templates/quota-exceeded' '0.0.0.0 25'
  _send_email 'email-templates/quota-exceeded' '0.0.0.0 25'
```